src/CMakeLists.txt

```diff
@@ -66,7 +67,7 @@ set(WERROR_FLAGS "-Werror -Wno-zero-length-array -Wno-c99-extensions          \
                   -Wno-zero-length-array")
 
 set(COMMON_FLAGS "-D__STDC_FORMAT_MACROS -D_GNU_SOURCE -DNEED_CPU_H  -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -DTARGET_PHYS_ADDR_BITS=64")
-set(COMMON_FLAGS "${COMMON_FLAGS} -Wall -fPIC -fno-strict-aliasing -fexceptions")
+set(COMMON_FLAGS "${COMMON_FLAGS} -Wall -O0 -fPIC -fno-strict-aliasing -fexceptions")
```

Comment:
Is there any reason for -O0?

Reply:
I just used it for debug, I will remove it.